main.go

```diff
@@ -39,6 +36,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
+	"github.com/go-logr/logr"
+	"github.com/jackc/pgx/v4"
```

Comment:
nit: third party package imports should go in the second grouping.

Reply:
This was added by Ajinkya. I will let him know and confirm this.

Reply:
Resolved